modules/rhizo_base/manifests/init.pp

```diff
@@ -299,7 +299,7 @@
       schedule => 'always',
       ensure   => latest,
       provider => git,
-      source   => 'git@dev.rhizomatica.org:rhizomatica/rccn.git',
+      source   => 'https://github.com/Rhizomatica/rccn.git',
```

Comment:
Ah, Yes, in this public release it should ref the public repo. Maybe we should make this a hiera parameter?

Reply:
That makes sense. If there's a private version it should probably be parameterized. Will do!